src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -0,0 +1,1497 @@
+            {
+                if (SelectedDebugProfile != null && SelectedDebugProfile.WorkingDirectory != value)
+                {
+                    SelectedDebugProfile.WorkingDirectory = value;
+                    OnPropertyChanged(nameof(WorkingDirectory));
+                }
+            }
+        }
+
+        public bool HasLaunchOption
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return SelectedDebugProfile.LaunchBrowser;
+            }
+            set
+            {
+                if (SelectedDebugProfile != null && SelectedDebugProfile.LaunchBrowser != value)
+                {
+                    SelectedDebugProfile.LaunchBrowser = value;
+                    OnPropertyChanged(nameof(HasLaunchOption));
+                }
+            }
+        }
+
+        public bool IsExecutable
+        {
+            get
+            {
+                if (SelectedLaunchType == null)
+                {
+                    return false;
+                }
+
+                return SelectedLaunchType.Kind == ProfileKind.Executable;
+            }
+        }
+
+        public bool IsProject
+        {
+            get
+            {
+                if (SelectedLaunchType == null)
+                {
+                    return false;
+                }
+
+                return SelectedLaunchType.Kind == ProfileKind.Project;
+            }
+        }
+
+        public bool IsCommand
+        {
+            get
+            {
+                if (SelectedLaunchType == null)
+                {
+                    return false;
+                }
+
+                return SelectedLaunchType.Kind == ProfileKind.BuiltInCommand || SelectedLaunchType.Kind == ProfileKind.CustomizedCommand;
+            }
+        }
+
+        public bool IsProfileSelected
+        {
+            get
+            {
+                return SelectedDebugProfile != null;
+            }
+        }
+
+        public bool IsIISExpress
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return SelectedDebugProfile.Kind == ProfileKind.IISExpress;
+            }
+        }
+
+        public bool IsIISOrIISExpress
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return SelectedDebugProfile.Kind == ProfileKind.IIS || SelectedDebugProfile.Kind == ProfileKind.IISExpress;
+            }
+        }
+
+        public bool IsWebProfile
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return IsIISOrIISExpress;
+            }
+        }
+
+        public bool ShowVersionSelector
```

Comment:
>ShowVersionSelector [](start = 20, length = 19)

Remove anything related to Versions. This was the DNX sdk version which never moved forward to dotnet in preview2.